.github/workflows/ci.yaml

```diff
@@ -8,44 +8,34 @@ on:
     branches:
       - main
   workflow_dispatch:
-  workflow_call:  # Allows the workflow to be called by other workflows
+  workflow_call:
```

Comment:
Wondering why removing the comment but doesn't matter. Makes sense to split up the matrix for the battery for sure.